### ISSUE TITLE
IPv6 and Sentinel

### ISSUE THREAD

**Issue by svantreeck** [microsoftarchive/redis#162]
Hello, I'm currently running a 2 computer cluster on the latest 2.8.12 with sentinels and running into some issues.

Computer A:
Sentinel 26379
Redis 6379 (master)

Computer B:
Sentinel 26379
Redis 6379 (slave)

If I set the sentinels to monitor 127.0.0.1 6379, it ends up telling the slave (computer B) to try to connect to 127.0.0.1 rather than computer A's IP or Hostname.  If I set the sentinels to monitor by hostname instead of 127.0.0.1, it looks like the sentinel sets up with an IPv6 address, and doesn't seem to establish correctly.  The best solution seems to be to use the computer's IPs directly instead of hostname or 127.0.0.1; however, I think it would be nice if using the hostname did not resolve to the IPv6 address.

```
sentinel masters output:
master0:name=mymaster,status=sdown,address=fe80::d8af:9b14:d120:e089:8379,slaves=0,sentinels=1
```

**Comment by jepickett** [comment]
This might be fixed with the changes included with 2.8.13. Try it out and let me know.

**Comment by svantreeck** [comment]
It looks like I'm still getting the same sdown status in 2.8.13; however, I'm seeing that they have some changes in 2.8.14 related to ipv6 and sentinel, so I'll wait until that lands and see if those remedy the problem.

**Comment by svantreeck** [comment]
BTW, this looks to be fixed in 2.8.14...it resolves hosts to ipv6 and is able to connect succesfully
